src/services/cronJobs/notifyDonationsWithSegment.ts

```diff
@@ -35,6 +35,9 @@ export const notifyMissingDonationsWithSegment = async () => {
     );
     const project = await Project.findOne({ id: donation.projectId });
     const owner = await User.findOne({ id: Number(project?.admin) });
+    // Notify owner donation was received ( if user and owner and project exist)
+    donation.segmentNotified = true;
```

Comment:
I think we move this update out of `if` scope, because if either project or user doesn't exist, it's not our problem, we set `segmentNotified‍‍‍‍` anyway, to don't check it again

Reply:
That's right, we tried our best to notify but if its not posible its not posible. Best effort service.